tests/functional/common.js

```diff
@@ -54,31 +55,23 @@ export function withTempAddonDir(
 }
 
 
-// reportRunCommandError helper
+// reportCommandErrors helper
 
-export function reportRunCommandError(obj: Object, msg: ?string) {
+export function reportCommandErrors(obj: Object, msg: ?string) {
   const errorMessage = msg || 'Unexpected web-ext functional test result';
-  const formattedErrorData = JSON.stringify(obj, null, 2);
+  const formattedErrorData = prettyjson.render(obj).replace('\\n', '\n');
```

Comment:
Now that you have it working, is this actually needed: `.replace('\\n', '\n')`? If I remove it, I don't see any difference.